.idea/codeStyles/codeStyleConfig.xml

```diff
@@ -1,5 +1,6 @@
 <component name="ProjectCodeStyleConfiguration">
   <state>
     <option name="USE_PER_PROJECT_SETTINGS" value="true" />
+    <option name="PREFERRED_PROJECT_CODE_STYLE" value="Default" />
```

Comment:
Not blocking for me, IDEA always tries to add it, but @avdyushin does this affect your code style settings somehow? Or is it just ignored? Does Android Studio try to add it back in again?

Reply:
Sorry, I didn't notice. Removed it

Reply:
Well, we should use code style from one attached to the project itself (I hope use per project stays for). But I have no idea why IDEA/AS keep updating settings back to default